ConsoleView.cpp

```diff
@@ -5,7 +5,7 @@
 //Prtint class will be used to print the name of a class given a UMLClass object
 //By using the get_class_name function from the UMLClass, it then prints the name of the Class given
 //@Para: UMLClass OBJ
-PrintClassName(UMLClass X){
+void ConsoleView::PrintClassName(UMLClass X){
```

Comment:
The method names still need to follow naming conventio (print_class_name) and make sure to change it in the header as well

Reply:
after adding the ConsoleView:: in the header files i got a extra qualification error

Reply:
Header files dont need the ConsoleView:: qualifier. Just the cpp file.